src/fsm/parser.c

```diff
@@ -1256,11 +1256,12 @@ ZL1:;
 		ADVANCE_LEXER; /* XXX: what if the first token is unrecognised? */
 		p_fsm(new, lex_state, act_state);
 
+		free(act_state_s.states.buckets);
```

Comment:
Explicit NULL; avoid use-after-free potential.

Reply:
`parser.c` is generated from `parser.act`.